kori/app/services/product.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from uuid import UUID
+
+import kori.app.dao.product as product_dao
+from kori.app.schemas.product import ProductCreateRequest, ProductSchema, ProductCreate, ProductUpdate
+
+
+def create(org_id: UUID, product_data: ProductCreateRequest) -> ProductSchema:
```

Comment:
If the service layer is going to be this simple, no harm is skipping it and just directly interact with DAO until the code has enough logic for a service layer.

No need to change it now, just mentioning as a bit of info.

Reply:
Noted